lib/components/Input/base/base.scss

```diff
@@ -0,0 +1,350 @@
+// @import "../../../styles/quill/spacing.scss";
+@import "@quill/spacing.scss";
+@import "@quill/font.size.scss";
+@import "@quill/color.scss";
+@import "@quill/color.semantic.scss";
+@import "@quill/color.opacity.scss";
+@import "@quill/motion.scss";
+@import "@quill/font.weight.scss";
+@import "@quill/border.scss";
+@import "@quill/line.height.scss";
+@import "@quill/size.scss";
+
+@keyframes dropIn {
+    from {
+        transform: translateY(-50%);
+    }
+    to {
+        transform: translateY(0);
+    }
+}
+
+.quill-input {
+    &__container {
+        display: flex;
+        width: 100%;
+        flex-direction: column;
+
+        .message__container {
+            display: flex;
+            justify-content: space-between;
+            padding-top: var(--semantic-spacing-general-sm);
+            font-family: "IBM Plex Sans";
+
+            &__text {
+                padding-block: var(--semantic-spacing-general-none);
+                padding-inline: var(--semantic-spacing-general-md);
+                font-size: var(--core-fontSize-50);
+                line-height: var(--core-lineHeight-100);
+                font-weight: var(--core-fontWeight-regular);
+                animation-name: dropIn;
+                transition-timing-function: var(--core-motion-ease-400);
+                animation-duration: var(--core-motion-duration-200);
+
+                &__status {
+                    &--neutral {
+                        color: var(--core-color-opacity-black-400);
+                    }
+                    &--error {
+                        color: var(--core-color-solid-red-900);
+                    }
+                    &--success {
+                        color: var(--core-color-solid-green-900);
+                    }
+                }
+
+                &__disabled {
+                    color: var(--core-color-opacity-black-400);
+                }
+            }
+
+            .self-end {
+                align-self: flex-end;
+            }
+        }
+    }
+
+    &__wrapper {
+        width: 100%;
+        display: inline-flex;
+        align-items: center;
+        padding-block: var(--core-size-50);
+        border-radius: var(--core-borderRadius-400);
+        border-width: var(--borderWidth-xs);
+        font-family: "IBM Plex Sans";
+        overflow: hidden;
+        position: relative;
+        cursor: pointer;
+        border-style: solid;
+        border-width: 1px;
+
+        &:has(input:disabled) {
```

Comment:
Could you please check if [:has](url) is working in all browsers? I remember in my case it wasn't working in mozilla even it has 92.11% of support in CanIUse

Reply:
okay. will check this

Reply:
as I checked, it's working in mozilla

Reply:
Thank you)